packages/destination-actions/src/destinations/mixpanel/incrementProperties/index.ts

```
@@ -0,0 +1,89 @@
+  defaultSubscription: 'type = "track"',
+  fields: {
+    ip: {
+      label: 'IP Address',
+      type: 'string',
+      description: "The IP address of the user. This is only used for geolocation and won't be stored.",
+      default: {
+        '@path': '$.context.ip'
+      }
+    },
+    user_id: {
+      label: 'User ID',
+      type: 'string',
+      allowNull: true,
+      description: 'The unique user identifier set by you',
+      default: {
+        '@path': '$.userId'
+      }
+    },
+    anonymous_id: {
+      label: 'Anonymous ID',
+      type: 'string',
+      allowNull: true,
+      description: 'The generated anonymous ID for the user',
+      default: {
+        '@path': '$.anonymousId'
+      }
+    },
+    increment: {
+      label: 'Increment Numerical Properties',
+      type: 'object',
+      description: 'Object of properties and the values to increment or decrement',
+      multiple: false,
+      defaultObjectUI: 'keyvalue',
+      default: {
+        '@path': '$.properties.increment'
+      }
+    }
+  },
+
+  perform: async (request, { payload, settings }) => {
+    if (!settings.projectToken) {
+      throw new IntegrationError('Missing project token', 'Missing required field', 400)
+    }
+
+    const apiServerUrl = getApiServerUrl(settings.apiRegion)
+
+    const responses = []
+
+    if (payload.increment && Object.keys(payload.increment).length > 0) {
+      const data: MixpanelEngageProperties = {
+        $token: settings.projectToken,
+        $distinct_id: payload.user_id ?? payload.anonymous_id,
+        $ip: payload.ip
+      }
+      data.$add = {}
+
+      for (const key of Object.keys(payload.increment)) {
+        const value = payload.increment[key]
+        if (typeof value === 'string' || typeof value === 'number') {
+          if (!isNaN(+value)) {
+            data.$add[key] = +value
+          }
+        }
+      }
```

Can we throw an error if any of the values are not numeric? Best to provide feedback to the customer by doing this.